src/pudl_archiver/archivers/validate.py

```diff
@@ -0,0 +1,35 @@
+"""Defines models used for validating/summarizing an archiver run."""
+from typing import Literal
+
+from pydantic import BaseModel
+
+
+class ValidationTestResult(BaseModel):
+    """Class containing results of a validation test, and metdata about the test."""
+
+    name: str
+    description: str
+    ignore_failure: bool = False
+    resource_name: str | None = None
+    result: bool
```

Comment:
I think `success` would give a clearer expectation of this being a bool, and what True vs. False would mean than `result`

Reply:
Agreed.